scripts/commands/rename.lua

```diff
@@ -0,0 +1,41 @@
+-----------------------------------
+-- func: !rename
+-- desc: Renames target NPC or MOB. Limited to 16 character spaces.
+--       Does not work on players. Does not alter database.
+-----------------------------------
+local commandObj = {}
+
+commandObj.cmdprops =
+{
+    permission = 1,
+    parameters = 'si'
+}
+
+local function error(player, msg)
+    player:PrintToPlayer(msg)
+    player:PrintToPlayer('!rename (string) (optional npcID/mobID)')
+end
+
+commandObj.onTrigger = function(player, inputString, entityID)
+    local entity = player:getCursorTarget()
+
+    if entityID then
+        entity = GetMobByID(entityID)
+        if not entity then
+            entity = GetNPCByID(entityID)
+        end
+    end
+
+    if not entity then
+        error(player, "Entity not found")
+        return
+    end
+
+    if inputString == '""' then
+        entity:renameEntity("", true)
```

Comment:
Gotta replace the `"` with an `'`

Reply:
_frownyface_

Reply:
I'm leaving the undo input as double quotes though, because I am NOT escaping single quotes inside single quotes for this convention.